os/binfmt/binfmt_arch_apis.c

```diff
@@ -109,19 +109,19 @@ void binfmt_arch_init_mem_protect(struct binary_s *binp)
 
 void binfmt_arch_deinit_mem_protect(struct binary_s *binp)
 {
-#if (defined(CONFIG_ARMV7M_MPU) || defined(CONFIG_ARMV8M_MPU))
 #ifdef CONFIG_SUPPORT_COMMON_BINARY
 	if (binp->islibrary) {
+#if (defined(CONFIG_ARMV7M_MPU) || defined(CONFIG_ARMV8M_MPU))
```

Comment:
os/kernel/task/task_terminate_unloaded.c
os/kernel/task/task_terminate.c

Both uses `CONFIG_ARM_MPU` instead of 7m_mpu and 8m_mpu configs.
https://github.com/Samsung/TizenRT/pull/5948/files#diff-de18ff5621c82ce73f2ccd2b0d9bbead6e92ae7836171469f03d4610b0cd6d34R191
Do we need to use them differently?